### ISSUE TITLE
Alert Pop Up Styling

### ISSUE THREAD

**Issue by fredochieng** [realrashid/sweet-alert#33]
- Package Version: #.#.#
- Laravel Version: #.#.#

### Description:
Hello, how do I style the alert? My alerts pop ups are too small

### Steps To Reproduce:

**Comment by realrashid** [comment]
@fredochieng right now its not possible to change the size of modal window but in next update you will get  ``` size():  ``` or  ``` width(): ``` helper method and some more helper methods too.

**Comment by fredochieng** [comment]
Okay

**Comment by timomch2** [comment]
Can you also include the ability to reduce or increase popup time in that
update please?

**Comment by realrashid** [comment]
@timomch2  It's already there!
##### AutoClose Alert

`
autoClose($milliseconds = 5000)
`
``` php
// example:
alert('Title','Lorem Lorem Lorem', 'success')->autoClose(5000);
```

**Comment by realrashid** [comment]
Fix #33
